README.md

```diff
@@ -1,95 +1,133 @@
+This package allows users to include additional columns to the source task table.  To do this, include any additional columns to the `asana_source` pass-through variables to ensure the downstream columns are present.
 
 ```yml
-# dbt_project.yml
+vars:
+  asana_source:
+    task_pass_through_columns: [custom_status, custom_department]
+```
 
-...
+### Changing the Build Schema
+By default this package will build the Asana staging models within a schema titled (<target_schema> + `_stg_asana`) and the Asana final models with a schema titled (<target_schema> + `_asana`) in your target database. If this is not where you would like your modeled Asana data to be written to, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
 models:
   asana:
     +schema: my_new_schema_name # leave blank for just the target_schema
   asana_source:
     +schema: my_new_schema_name # leave blank for just the target_schema
 ```
+</details>
 
-This package allows users to include additional columns to the source task table.  To do this, include any additional columns to the `asana_source` pass-through variables to ensure the downstream columns are present.
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
 
-```yml
-# dbt_project.yml
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_asana_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
 
-...
+```yml
 vars:
-  asana_source:
-    task_pass_through_columns: [custom_status, custom_department]
+    asana_<default_source_table_name>_identifier: your_table_name 
 ```
 
-## Database support
-This package is compatible with BigQuery, Snowflake, and Redshift.
+</details>
+
+## (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™    
+<details><summary>Expand for more details</summary>
+
+Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
+
+</details>
+
+# 🔍 Does this package have dependencies?
+This dbt package is dependent on the following dbt packages. Please be aware that these dependencies are installed by default within this package. For more information on the following packages, refer to the [dbt hub](https://hub.getdbt.com/) site.
+> IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
+
+```yml
+packages:
+    - package: fivetran/asana_source
+      version: [">=0.5.0", "<0.6.0"]
```

Comment:
We will need to update this range as well since the source package will be getting a breaking change.

Reply:
updated